sdk/dapp-kit/test/components/SuiClientProvider.test.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { act, render, screen } from '@testing-library/react';
+import { SuiClientProvider } from '../../src/components/SuiClientProvider.js';
+import { useSuiClient, useSuiClientContext } from 'dapp-kit/src/index.js';
+import { SuiClient } from '@mysten/sui.js/client';
+import { useState } from 'react';
+
+describe('SuiClientProvider', () => {
+	it('renders without crashing', () => {
+		render(
+			<SuiClientProvider>
+				<div>Test</div>
+			</SuiClientProvider>,
+		);
+		expect(screen.getByText('Test').textContent).toEqual('Test');
```

Comment:
small nit: I think adding and using `@testing-library/jest-dom` would be a nice touch since we're going to be writing more and more component tests eventually. With this particular case, `expect(screen.getByText('Test')).toBeInTheDocument()` would be a little more concise and to the point than having to write `Test` twice 😄

Reply:
got this set up!